src/messages.js

```diff
@@ -1,9 +1,11 @@
 export const MESSAGE = {
     browserDisconnected:            'The {userAgent} browser disconnected. This problem may appear when a browser hangs or is closed, or due to network issues.',
+    browsersDisconnected:           'The following browsers disconnected: {userAgents}. This problem may appear when a browser hangs or is closed, or due to network issues.',
```

Comment:
okay then, the final (and checked) version:
`cantRunAgainstDisconnectedBrowsers: 'The following browsers disconnected: {userAgents}. Tests won't be run.',`